SolizWilson/VentaDeRepuestos/src/cliente.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class cliente {
```

Comment:
Okay, I see what are you adding in here, this is not what I meant when I said, let's first focus on the back-end of your application. For example, in this class client there is no need for a main method. This is class is the abstraction of a customer, so in here you will only have some attributes and their getters and setters. nothing else. Then on the class Main on the method main there you will create a customer object based on the information obtained from the scanner. Something like this:
public static void maint(String[] args) {
    System.out.printf("BIENVENIDO AL SISTEMA DE REPUESTOS");
    System.out.printf("Que desea hacer? Presione 1 si desea registrar un cliente.\n Presione 2 si desea registrar un producto");
    Scanner reader = new Scanner((System.in));
    if (reader.nextInt() == 1) {
        Customer customerToRegister = new Customer();
        System.out.printf("Introduzca el nombre del cliente");
        customerToRegister.setName(reader.nextLine());
        System.out.printf("Introduzca el numero de telefono del cliente");      
        customerToRegister.setPhoneNumber(reader.nextLine());
    }
}
And So on the main method inside Main class will be the menu but you will creating object based on the classes that you create.